protocol/x/clob/keeper/rate_limit.go

```diff
@@ -31,6 +36,11 @@ func (k *Keeper) RateLimitCancelOrder(ctx sdk.Context, msg *types.MsgCancelOrder
 
 // RateLimitPlaceOrder passes orders with valid clob pairs to `placeOrderRateLimiter`.
 func (k *Keeper) RateLimitPlaceOrder(ctx sdk.Context, msg *types.MsgPlaceOrder) error {
+	// Only rate limit during `CheckTx` and `ReCheckTx`
```

Comment:
So for now we'll also skipping rate-limiting for stateful orders in `DeliverTx`?

If this is intended, mind creating a ticket to re-enable it? Can be M6 or later

Also if possible note in func comment

Reply:
Disabling rate limiting was brought up in slack [here](https://dydx-team.slack.com/archives/C05233L8E5A/p1696462672390229) and [was proposed to be the long term solution here](https://dydx-team.slack.com/archives/C04BNGM82HM/p1696456267230799).

The work to disable rate limiting was tracked in https://linear.app/dydx/issue/CLOB-937